src/content/enterprise/index.md

```diff
@@ -66,7 +66,7 @@ Some collaborative efforts to make Ethereum enterprise friendly have been made b
 
 ### Tooling and libraries {#tooling-and-libraries}
 
-- [Alethio](https://aleth.io/) _Ethereum Data Analytics Platform_
+- [Aleth](https://github.com/ethereum/aleth) _Ethereum Data Analytics Platform_
```

Comment:
This is a different project. Aleth is an Ethereum client.

I've pinged Alethio to see if the project is still active: https://twitter.com/samonchain/status/1359307457588207621?s=20

Reply:
Thanks for this. I've left a message on their [discord](https://discordapp.com/channels/608940867222372353/608940867222372357/808889052312240128) server too (haven't heard back yet).  However, it seems like https://aleth.io is the official url, even though it's down for now. It's the site listed on their their [twitter](https://twitter.com/AlethioEthstats) profile too. So I will revert the URL back to https://aleth.io/.